go-console/.conf/runContainerIfNotExists.ps1

```diff
@@ -0,0 +1,21 @@
+
+$_containerRuntime = $args[0]
```

Comment:
This file can be removed, this is automatically copied from scripts to the project folder during the creation.